presto-main/src/main/java/com/facebook/presto/server/protocol/QueuedStatementResource.java

```diff
@@ -118,14 +120,18 @@
     private final boolean compressionEnabled;
 
     private final SqlParserOptions sqlParserOptions;
+    private final TracerProvider tracerProvider;
+    private final SessionPropertyManager sessionPropertyManager;     // We may need some system default session property values at early query stage even before session is created.
 
     @Inject
     public QueuedStatementResource(
             DispatchManager dispatchManager,
             DispatchExecutor executor,
             LocalQueryProvider queryResultsProvider,
             SqlParserOptions sqlParserOptions,
-            ServerConfig serverConfig)
+            ServerConfig serverConfig,
+            TracerProvider tracerProvider,
+            SessionPropertyManager sessionPropertyManager)
```

Comment:
how do these get initialized? not too familiar with DI, but I don't see any other changes that would update any default values for specifically the tracerProvider.